packages/devtools_app/lib/src/screens/inspector/inspector_screen_details_tab.dart

```diff
@@ -122,14 +122,10 @@ class _InspectorExpandCollapseButtonsState
   }
 
   void _onCollapseClick() {
-    unawaited(
-      blockWhileInProgress(() async {
-        ga.select(
-          gac.inspector,
-          gac.collapseAll,
-        );
-        await widget.controller.collapseDetailsToSelected();
-      }),
+    ga.select(
```

Comment:
why do we no longer want to blockWhileInProgress?

Reply:
@incendial this is an example of a good success story for the Dart Code Metrics lints. The lints caught that ga.select never needed to be async which triggered further lints to clarify that we never needed to be calling `blockWhileInProgress` as nothing async was actually happening in the whole block.